tools/mwa/mwa_astro_tool.xml

```diff
@@ -0,0 +1,117 @@
+    </data>
+  </outputs>
+  <tests>
+    <test expect_num_outputs="2">
+      <conditional name="_data_product">
+        <param name="_selector" value="Image" />
+        <param name="src_name" value="1ES 0229+200" />
+        <param name="RA" value="38.202562" />
+        <param name="DEC" value="20.288191" />
+        <param name="T1" value="2000-10-09T13:16:00.0" />
+        <param name="T2" value="2022-10-10T13:16:00.0" />
+        <param name="Radius" value="1.0" />
+        <param name="pixsize" value="0.01" />
+        <param name="Frequency" value="GLEAM 170-231 MHz" />
+      </conditional>
+      <assert_stdout>
+        <has_text text="*** Job finished successfully ***" />
+      </assert_stdout>
+    </test>
+    <test expect_num_outputs="2">
+      <conditional name="_data_product">
+        <param name="_selector" value="Spectrum" />
+        <param name="src_name" value="1ES 0229+200" />
+        <param name="RA" value="38.202562" />
+        <param name="DEC" value="20.288191" />
+        <param name="T1" value="2000-10-09T13:16:00.0" />
+        <param name="T2" value="2022-10-10T13:16:00.0" />
+        <param name="Radius" value="0.05" />
+      </conditional>
+      <assert_stdout>
+        <has_text text="*** Job finished successfully ***" />
+      </assert_stdout>
+    </test>
+  </tests>
+  <help>This is help for MWA
+</help>
+  <citations>
+    <citation type="bibtex">@article{2017MNRAS.464.1146H,
+			author = {{Hurley-Walker}, N. and {Callingham}, J.~R. and {Hancock}, P.~J. and {Franzen}, T.~M.~O. and {Hindson}, L. and {Kapi{\'n}ska}, A.~D. and {Morgan}, J. and {Offringa}, A.~R. and {Wayth}, R.~B. and {Wu}, C. and {Zheng}, Q. and {Murphy}, T. and {Bell}, M.~E. and {Dwarakanath}, K.~S. and {For}, B. and {Gaensler}, B.~M. and {Johnston-Hollitt}, M. and {Lenc}, E. and {Procopio}, P. and {Staveley-Smith}, L. and {Ekers}, R. and {Bowman}, J.~D. and {Briggs}, F. and {Cappallo}, R.~J. and {Deshpande}, A.~A. and {Greenhill}, L. and {Hazelton}, B.~J. and {Kaplan}, D.~L. and {Lonsdale}, C.~J. and {McWhirter}, S.~R. and {Mitchell}, D.~A. and {Morales}, M.~F. and {Morgan}, E. and {Oberoi}, D. and {Ord}, S.~M. and {Prabu}, T. and {Shankar}, N. Udaya and {Srivani}, K.~S. and {Subrahmanyan}, R. and {Tingay}, S.~J. and {Webster}, R.~L. and {Williams}, A. and {Williams}, C.~L.},
+			title = {{GaLactic and Extragalactic All-sky Murchison Widefield Array (GLEAM) survey - I. A low-frequency extragalactic catalogue}},
+			journal = {\mnras},
+			keywords = {techniques: interferometric, surveys, galaxies: general, Astrophysics - Astrophysics of Galaxies},
+			year = {2017},
+			month = {jan},
+			volume = {464},
+			number = {1},
+			pages = {1146-1167},
+			doi = {10.1093/mnras/stw2337},
+			archivePrefix = {arXiv},
+			eprint = {1610.08318},
+			primaryClass = {astro-ph.GA},
+			adsurl = {https://ui.adsabs.harvard.edu/abs/2017MNRAS.464.1146H},
+			adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+		}</citation>
```

Comment:
```suggestion
    <citation type="doi">10.1093/mnras/stw2337</citation>
```

Reply:
I imagined this is a preferred way. The bot is just currently only doing dummy copying. This will improve.